### PR TITLE
Add status theo

### PR DESCRIPTION
**Pull request by ruromero**
Rebased and reviewed version of #102

### CODE REVIEW

File: mercury-operator/src/main/java/com/redhat/mercury/operator/utils/ResourceUtils.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+public class ResourceUtils {
+
+    public static final String capitalize(Boolean value) {
+        if (Boolean.TRUE.equals(value)) {
+            return "True";
+        }
+        return "False";
+    }
+
```
Comment (tmihalac) [inline mercury-operator/src/main/java/com/redhat/mercury/operator/utils/ResourceUtils.java:11]: please add:

    public static String now() {
        return ZonedDateTime.now(ZoneOffset.UTC).format(DateTimeFormatter.ISO_INSTANT);
    }

Comment (ruromero) [reply mercury-operator/src/main/java/com/redhat/mercury/operator/utils/ResourceUtils.java:11]: where is it used?

File: mercury-operator/src/main/java/com/redhat/mercury/operator/controller/AbstractController.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                .build());
+    }
+
+    protected void setStatusCondition(T resource, Condition condition) {
+        if (resource.getStatus() == null) {
+            return;
+        }
+        Condition current = resource.getStatus().getCondition(condition.getType());
+        if (current != null) {
+            condition.setLastTransitionTime(current.getLastTransitionTime());
```
Comment (tmihalac) [inline mercury-operator/src/main/java/com/redhat/mercury/operator/controller/AbstractController.java:78]: please change it to ResourceUtils.now()

File: mercury-operator/src/main/java/com/redhat/mercury/operator/controller/AbstractController.java [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        }
+        if (!condition.equals(current)) {
+            condition.setLastTransitionTime(new Date().toString());
```
Comment (tmihalac) [inline mercury-operator/src/main/java/com/redhat/mercury/operator/controller/AbstractController.java:81]: please change it to ResourceUtils.now()

File: mercury-operator/src/main/java/com/redhat/mercury/operator/utils/ResourceUtils.java (hunk 1 above)
Comment (tmihalac) [reply mercury-operator/src/main/java/com/redhat/mercury/operator/utils/ResourceUtils.java:11]: Please use it here condition.setLastTransitionTime(new Date().toString());

**Pull request merged**